scripts/OfekQuery.js

```diff
@@ -0,0 +1,288 @@
+function $(name) {
+
+	return new ofekQuery(name);
+}/**/
+var ofekQuery = function (query) {
+	this.elements =[];
+
+	if (query.split(" ").length > 1)
+	{
+        var query = query.split(" ");
+        var potentialElements = [];
+        var potentialPElements= [];
+        var elementsArray = [];
+        var deleteArr= [];
+
+        var last = query[query.length - 1];
+
+
+        if (last.match(/^\..*/i)){
+            elementsArray = document.getElementsByClassName(last.substring(1, last.length));
+        }
+        else if (last.match(/^#.*/i)) {
+            elementsArray = document.getElementById(last.substring(1, last.length));
+        }
+        else if (last.match(/^\w.*/i)){
+            elementsArray = document.getElementsByTagName(last);
+        }
+
+        for (var qIndex = query.length - 2; qIndex >= 0 ; qIndex--){
```

Comment:
Basically you have functions and it's not a shame to use them.
Split to functions so I wont have to read this whole horrible code